framework/src/media/MediaPlayerImpl.h

```diff
@@ -63,7 +63,7 @@ class MediaPlayerImpl : public std::enable_shared_from_this<MediaPlayerImpl>
 	player_result_t stop();
 
 	int getVolume();
```

Comment:
if you want to setVolume dataType is uint8_t of parameter, 
this api also return type will be uint8_t

Reply:
I think so, too.
But let's modify this in another PR, I'll mark ToDo here.